taylorseager/INDIVIDUAL-ASSIGNMENT-Team-Roster
# setup pages complete

## Description
All of the setup pages 

## Related Issue
Please link to the issue here: #2 #3 #4 #6 #7 #22 #10 #14 #8 

## Motivation and Context
These files/visual representations are necessary to the planning process to hopefully help the coding process go more smoothly.

## How Can This Be Tested?
running the server locally on your machine; you should see the navBar updates, sign in/sign out pages/buttons, and the navBar routes posted.

## Types of changes

- [ x] New feature (non-breaking change which adds functionality)
